packages/graphql/tests/integration/issues/4532.int.test.ts

```diff
@@ -17,25 +17,22 @@
  * limitations under the License.
  */
 
-import { graphql } from "graphql";
-import { type Driver } from "neo4j-driver";
-import { Neo4jGraphQL } from "../../../src/classes";
-import { cleanNodesUsingSession } from "../../utils/clean-nodes";
-import { UniqueType } from "../../utils/graphql-types";
-import { default as Neo4jHelper } from "../neo4j";
+import type { UniqueType } from "../../utils/graphql-types";
+import { TestHelper } from "../utils/tests-helper";
 
 describe("https://github.com/neo4j/graphql/issues/4532", () => {
-    let driver: Driver;
-    let neo4j: Neo4jHelper;
-    let neo4jGraphql: Neo4jGraphQL;
+    let testHelper: TestHelper;
```

Comment:
in this case I believe that having the testHelper defined here may cause some flakyness as two describe below have different typeDefs

Reply:
mm, it should be fine, as the typedefs are set in initNeo4jGraphQL and there is a hook throwing an error if you try to call this method twice on the same  testHelper. The different describes in the same file are executed in series